force-app/main/default/classes/OpenWeatherCheckerTest.cls

```diff
@@ -0,0 +1,90 @@
+/**
+ * File:        OpenWeatherCheckerTest.cls
+ * Project:     OSFPracticalExercises
+ * Date:        August 19, 2021
+ * Created By:  Can Yavuzkurt
+ * *************************************************************************
+ * Description:  Unit test for OpenWeatherChecker.cls
+ * *************************************************************************
+ * History:
+ * Date:                Modified By:                         Description:
+ * August 19, 2021      Can Yavuzkurt                        Created
+ */
+@IsTest
+private class OpenWeatherCheckerTest {
```

Comment:
Please create 2 "mock" classes, 1 for success, 2nd for bad request

Reply:
Updated